articles/iot-edge/how-to-store-data-blob.md

```diff
@@ -157,6 +157,14 @@ sudo chmod -R 700 <blob-dir>
 
 For information on configuring log files for your module, see these [production best practices](./production-checklist.md#set-up-logs-and-diagnostics).
 
+To configure the output log level, you can set `LogLevel` environment variable for this module. `LogLevel` can take the following values (the default value is set to `Info`):
+
+* Critical
+* Error
+* Warning
+* Info
+* Debug
+
```

Comment:
```suggestion
The default output log level is 'Info'.  To change the output log level, Set the `LogLevel` environment variable for this module in the deployment manifest. `LogLevel` accepts the following values: 

* Critical
* Error
* Warning
* Info
* Debug

For information on configuring log files for your module, see these [production best practices](./production-checklist.md#set-up-logs-and-diagnostics).
```

Reply:
Let's put the *For more information...` link at the bottom of the section. I reworded it a little and mentioned the deployment manifest.